.github/workflows/review-quarantine-pr.yml

```diff
@@ -0,0 +1,23 @@
+name: Add maintainers of quarantined test scenarios to PR
```

Comment:
this is what shows on a side tab when looking at a workflow. Has to be short. https://github.com/katgiadla/sdk-nrf/actions/workflows/review-quarantine-pr.yml
E.g. Quarantine validation